etc/kayobe/dnf.yml

```diff
@@ -135,6 +140,16 @@ dnf_custom_repos_el9:
     gpgkey: "{{ dnf_docker_gpg_key_url }}"
     gpgcheck: yes
 
+# ELRepo 9
+dnf_custom_repos_elrepo_9:
```

Comment:
Added this as a separate top-level dict to avoid making the repofile on the majority of systems that don't have hardware which requires elrepo rpms